packages/web-components/fast-foundation/src/combobox/combobox.ts

```diff
@@ -568,12 +572,14 @@ export class FASTCombobox extends FormAssociatedCombobox {
     public setDefaultSelectedOption(): void {
         if (this.$fastController.isConnected && this.options) {
             const selectedIndex = this.options.findIndex(
-                el => el.getAttribute("selected") !== null || el.selected
+                el => el.hasAttribute("selected") || el.selected
```

Comment:
Does this address a bug?

Reply:
I was doing a lot with boolean attributes while writing the playwright extensions `toHaveBooleanAttribute` and `hasAttribute` and I guess this particular change slipped through the cracks.